site/src/atomics/display.md

```diff
@@ -0,0 +1,79 @@
+---
+title: Display Atomics
+description: Display related atomic css classes for the Atlas Design System
+template: standard
+---
+
+# Display Atomics
+
+The following displays are covered: `block`, `flex`, `inline`, `inline-block`, `inline-flex`, and `none`.
+
+The following breakpoints are supported for each display value:
+
+```Text
+.position-{value}-tablet
+.position-{value}-desktop
+.position-{value}-widescreen
+```
+
+## Block
```

Comment:
Do we need six different code examples to explain one pattern?

Reply:
Reduced to one.